zerver/lib/message.py

```diff
@@ -320,6 +322,7 @@ def render_markdown(message, content, realm=None, realm_alert_words=None, messag
             if user_id in message_user_ids:
                 possible_words.update(set(words))
 
+    available_slash_commands = set(get_slash_commands_by_realm(realm, is_active=True))
```

Comment:
I meant two separate things here.

(1) Instead of retrieving `available_slash_commands` here, can you please retrieve it in the same place that `realm_alert_words` is retrieved, and then handle it the same way as `realm_alert_words` (i.e. by passing it in as a parameter)?  This way it will be clear that alert words and slash commands are handled very similarly.

(2) It might be the case that `realm_slash_commands` is a clearer name than `available_slash_commands`, but renaming this is less important than (1).